exercise02/exercises01.js

```diff
@@ -0,0 +1,5 @@
+function calculateAverage(numbers) {
+    const sum = numbers.reduce((previousNumber, currentNumber) => previousNumber + currentNumber);
+    const count = numbers.length;
+    return sum / count;
```

Comment:
I would say that `numbers.length` is very simple and understandable and it totally fine to have it in the return type and get rid of `count`
`return sum / numbers.length`